easybuild/easyblocks/t/trinity.py

```diff
@@ -114,6 +115,8 @@ def inchworm(self, run=True):
         """Install procedure for Inchworm."""
 
         make_flags = 'CXXFLAGS="%s %s"' % (os.getenv('CXXFLAGS'), self.toolchain.get_flag('openmp'))
+        if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
```

Comment:
you're comparing a `LooseVersion` instance with a string here

Python allows that, and it may *seem* to work, but it doesn't do what you think it does

please change to (note the `>=`):

```python
if LooseVersion(self.version) >= LooseVersion('2.0') and LooseVersion(self.version) < LooseVersion('3.0'):
```